resources/views/auth/application/application.blade.php

```diff
@@ -12,19 +12,43 @@
                 </div>
                 <div class="col s12 xl8">
                     @can('viewAnyApplication', \App\Models\User::class)
-                        <span class="right">
-                            @include('auth.application.status', ['status' => $user->application->status])
-                            @if(auth()->user()->hasRole(\App\Models\Role::SYS_ADMIN))
-                                @if($user->application->status == \App\Models\ApplicationForm::STATUS_SUBMITTED)
-                                    <form method="POST" style="display: inline" action="{{route('applications.edit')}}">
-                                        @csrf
-                                        <input type="hidden" name="application" value="{{$user->application->id}}"/>
-                                        <input type="hidden" name="banish" value="true"/>
-                                        <x-input.button class="red tooltipped btn-small" data-tooltip="Elutasít" floating icon="close"/>
-                                    </form>
-                                @endif
+                        @if(!auth()->user()->hasRole(\App\Models\Role::SYS_ADMIN) || $user->application->status == \App\Models\ApplicationForm::STATUS_IN_PROGRESS)
+                            <span class="right">
+                                @include('auth.application.status', ['status' => $user->application->status])
+                            </span>
+                        @else
+                            @if($user->application->status != \App\Models\ApplicationForm::STATUS_IN_PROGRESS)
+                            
+                            <form id="status-form-{{$user->id}}" method="POST" style="display: inline" action="{{route('applications.edit')}}">
```

Comment:
Here I think its better to add the application id to the route, not as a field.

Reply:
This is not there because we need it, but to make the form unique. There will be as many forms like this as there are applications displayed at once, and I needed a unique id.